include/device_queue.h

```diff
@@ -149,7 +149,7 @@ namespace detail {
 
 	template <typename DeviceT, typename PlatformT, typename SelectorT>
 	bool try_find_one_device(
-	    std::string& how_selected, DeviceT& device, const std::vector<PlatformT>& platforms, const host_config& host_cfg, SelectorT selector) {
+	    std::string& /*how_selected*/, DeviceT& device, const std::vector<PlatformT>& platforms, const host_config& /*host_cfg*/, SelectorT selector) {
```

Comment:
Since `how_selected` and `host_cfg` are actually unused, they should be removed here and at the callsite.